amazon/iontest/ion_test_driver_config.py

```diff
@@ -36,15 +37,21 @@ def install_ion_c(log):
     log_call(log, (TOOL_DEPENDENCIES['cmake'], '--build', '.'))
 
 
+def install_ion_java(log):
+    log_call(log, (TOOL_DEPENDENCIES['maven'], '-f', 'ion-java-cli/pom.xml', 'package'))
+
+
 ION_BUILDS = {
     'ion-c': IonBuild(install_ion_c, os.path.join('tools', 'cli', 'ion')),
     'ion-tests': NO_OP_BUILD,
+    'ion-java': IonBuild(install_ion_java, os.path.join('ion-java-cli', 'target', 'ion-java-cli-1.0.jar'))
     # TODO add more implementations here
 }
 
 # Ion implementations hosted in Github. Local implementations may be tested using the `--implementation` argument,
 # and should not be added here. For the proper description format, see the ion_test_driver CLI help.
 ION_IMPLEMENTATIONS = [
     'ion-c,https://github.com/amzn/ion-c.git,master',
+    'ion-java,git@github.com:cheqianh/ion-java.git,pr6'
```

Comment:
Remember to change this once we merge your ion-java changes to mainline.

Reply:
Sure.